app/styles/input.module.css

```diff
@@ -13,10 +17,20 @@
   outline: none;
   transition: all 0.1s;
   border-radius: 8px;
-  border: 1px solid var( --color-shade-grey);
+  border: 1px solid var(--color-shade-grey);
   background: var(--color-white);
 }
 
+.input-disable {
+  opacity: 1.5;
+  pointer-events: none;
+  color: #777;
+  background-color: #f2f2f2;
+  box-shadow: none;
+  cursor: not-allowed;
+  border: 1px solid #ccc;
```

Comment:
use variables for color names

Reply:
Done with changes can you please check my latest commit

https://github.com/Real-Dev-Squad/website-www/assets/107163260/b679a78c-3fcc-4d45-9623-198e5a24789d